test/functional/apps/visualize/_point_series_options.js

```diff
@@ -203,11 +204,87 @@ export default function ({ getService, getPageObjects }) {
 
```

Comment:
nit: The test suite is called `'x axis labels'`, but now it's also testing values. The common thing is correct behavior during timezone switches, so we could rename it `'timezones'` or something like that.

Reply:
agreed, changes coming in next push.